packages/kaizen-core/helpers/entity-fake-link/h-entity-fake-link.js

```diff
@@ -39,23 +41,43 @@ const fakeLinkProcessed = (el, link) => {
   });
 };
 
-export default ({
-  wrapperData = '[data-h-entity-fake-link-container]',
-  targetData = '[data-h-entity-fake-link-target]',
-  processingName = 'h-entity-fake-link',
-  context = document,
-} = {}) => {
-  once(processingName, wrapperData, context).forEach((el) => {
-    // Multiple links inside of element you want to wrap is not expected
-    // with this script. So it searches only the first matched.
-    let link = el.querySelector(`a${targetData}[href]`);
-    if (!link) {
-      // Fallback search. If targetData is not defined, then let's try
-      // to catch just first matched link.
-      link = el.querySelector('a[href]');
-    }
-    if (link) {
-      fakeLinkProcessed(el, link);
-    }
-  });
-};
\ No newline at end of file
+(({ behaviors }) => {
+  behaviors.kaizen_core_h_entity_fake_link = {
+    attach: (context, settings) => {
+      const customConfig =
+        settings &&
```

Comment:
maybe rewrite this with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator